commit 845879246b16890224d1c02dbb89d10561433e17
Author: Theodore Ts'o <tytso@mit.edu>
Date:   Sat Feb 4 23:14:19 2017 -0500

    jbd2: don't leak modified metadata buffers on an aborted journal
    
    commit e112666b4959b25a8552d63bc564e1059be703e8 upstream.
    
    If the journal has been aborted, we shouldn't mark the underlying
    buffer head as dirty, since that will cause the metadata block to get
    modified.  And if the journal has been aborted, we shouldn't allow
    this since it will almost certainly lead to a corrupted file system.
    
    Signed-off-by: Theodore Ts'o <tytso@mit.edu>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1273_845879246b16890224d1c02dbb89d10561433e17.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1273_845879246b16890224d1c02dbb89d10561433e17.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
